doc/source/cluster/cluster_under_construction/ray-clusters-on-kubernetes/examples/ml-example.md

```diff
@@ -1,10 +1,260 @@
+If you already know exactly how much compute your workload requires, it makes
+sense to provision a statically-sized Ray cluster.
+In this guide's example, we know that we need 1 Ray head and 9 Ray workers,
+so autoscaling is not strictly required.
+
+#### Longer end-to-end runtime.
+Autoscaling entails provisioning compute for Ray workers while the Ray application
+is running. On the other hand, if you pre-provision a fixed number of Ray nodes,
+all of the Ray nodes can be started in parallel, potentially reducing your application's
+runtime.
+
+## Kubernetes infrastructure setup
+
+### Managed Kubernetes services
+
+Running the example in this guide requires some basic Kubernetes infrastructure set-up.
+We collect some helpful links for users who are getting started with a managed Kubernetes service.
+
+#### Landing pages
+Landing pages for the major cloud providers' Kubernetes services can be found at the following links.
+- [GKE landing page (Google Cloud)](https://cloud.google.com/kubernetes-engine)
+- [EKS landing page (Amazon Web Services)](https://aws.amazon.com/eks/)
+- [AKS landing page (Azure)](https://azure.microsoft.com/en-us/services/kubernetes-service/)
+
+
+#### General documentation
+For documentation on managed Kubernetes services, see the following links.
+- [GKE docs](https://cloud.google.com/kubernetes-engine/docs/)
+- [EKS docs](https://docs.aws.amazon.com/eks/latest/userguide/)
+- [AKS docs](https://docs.microsoft.com/en-us/azure/aks/)
+
+#### Quickstart guides
+See the following links for guides on getting started with your managed Kubernetes service.
+- [GKE docs](https://cloud.google.com/kubernetes-engine/docs/deploy-app-cluster)
+- [EKS docs](https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html)
+- [AKS docs](https://docs.microsoft.com/en-us/azure/aks/learn/quick-kubernetes-deploy-portal?tabs=azure-cli)
+
+#### Setting up node pools
+Running the example workload in this guide requires setting up a pool or group of Kubernetes nodes.
+To learn about node group or node pool setup with managed Kubernetes services, see
+the following links.
+- [GKE node pools](https://cloud.google.com/kubernetes-engine/docs/concepts/node-pools)
+- [EKS node groups](https://docs.aws.amazon.com/eks/latest/userguide/managed-node-groups.html)
+- [AKS node pools](https://docs.microsoft.com/en-us/azure/aks/use-multiple-node-pools)
+
+### Set up a node pool for the XGBoost benchmark
+
+For the workload in this guide, it is recommended to use a pool or group of Kubernetes nodes
+with the following properties:
+- 10 nodes total
+- A capacity of 16 CPU and 64 Gi memory per node. For the major cloud providers, suitable instance types include
+    * m5.4xlarge (Amazon Web Services)
+    * Standard_D5_v2 (Azure)
+    * e2-standard-16 (Google Cloud)
+- Each node should be configured with 1000 gigabytes of disk space (to store the training set).
+
+### Optional: Set up an autoscaling node pool
```

Comment:
This can again be an infobox to mirror the infobox from above and make clear that it is optional.

Reply:
Moved mentions of autoscaling into blue boxes, with the word optional in prominent places.